src/video-interface.js

```diff
@@ -167,6 +183,19 @@ export const VideoAttributes = {
    * to the corner when scrolled out of view and has been interacted with.
    */
   DOCK: 'dock',
+  /**
+   * auto-fullscreen
+   *
+   * If enabled, this automatically expands the currently visible video and
+   * playing to fullscreen when the user changes the device's orientation to
+   * landscape if the video was started following a user interaction
+   * (not autoplay)
+   *
+   * Dependent upon browser support of
+   * http://caniuse.com/#feat=screen-orientation
+   * and http://caniuse.com/#feat=fullscreen
+   */
+  AUTOFULLSCREEN: 'auto-fullscreen',
```

Comment:
I am not a big fan of binary attributes. Let's do:
`fullscreen-on-landscape=(never|always)`. 
If `fullscreen-on-landscape` is specified without a value, then it equals `always`
If no `fullscreen-on-landscape` is specified, for initial launch, it means `never`, we will have to do an experiment and ramp up slowly where not having `fullscreen-on-landscape` starts meaning `always`